### ISSUE TITLE
China Frenzy Generals Power is very rarely used

### ISSUE THREAD

**Issue by xezon** [TheSuperHackers/GeneralsGamePatch#709]
China Frenzy Generals Power is very rarely used. Players typically prefer to pick the full Artillery Barrage. Check if it can be better and more attractive.

## Stats
```
WeaponBonus = FRENZY_ONE    DAMAGE 110%
WeaponBonus = FRENZY_TWO    DAMAGE 120%
WeaponBonus = FRENZY_THREE  DAMAGE 130%

SpecialPower SuperweaponFrenzy
  ReloadTime = 240000

Object Frenzy_InvisibleMarker_Level1
  BonusDuration = 10000
  BonusRange = 200

Object Frenzy_InvisibleMarker_Level2
  BonusDuration = 20000
  BonusRange = 200

Object Frenzy_InvisibleMarker_Level3
  BonusDuration = 30000
  BonusRange = 200
```

## Proposal 1
Decrease reload time.

## Proposal 2
Increase bonus duration.

## Proposal 3
Increase bonus range.

## Proposal 4
Increase bonus damage.

**Comment by commy2** [comment]
Voting for 1.

Also would consider **making the ability last 30 seconds regardless of which level it was upgraded**. Currently it increases damage and duration, so the ability as a whole becomes marginally useful once it is fully upgraded. The issue is that it is super weak before that.

**Comment by xezon** [comment]
One problem I have with this ability is the lack of immediate usefulness after cooldown. Unlike other abilities such as GPS Scrambler, Artillery Barrage, Spy Drone, et al, it may not be suitable to use straight away after power is ready to use. There may be no important battles going on yet. This will further increase its usage frequency and therefore decrease its impact on the match.

The same applies to the Emergency Repair power.

**Comment by xezon** [comment]
Proposed setup:

```
WeaponBonus = FRENZY_ONE    DAMAGE 110%
WeaponBonus = FRENZY_TWO    DAMAGE 120%
WeaponBonus = FRENZY_THREE  DAMAGE 130%

SpecialPower SuperweaponFrenzy
  ReloadTime = 180000

Object Frenzy_InvisibleMarker_Level1
  BonusDuration = 30000
  BonusRange = 200

Object Frenzy_InvisibleMarker_Level2
  BonusDuration = 30000
  BonusRange = 200

Object Frenzy_InvisibleMarker_Level3
  BonusDuration = 30000
  BonusRange = 200
```

**Comment by xezon** [comment]
Maybe could even boost the damage by 5 % or so. Though I have no data to show that would justify it.

```
WeaponBonus = FRENZY_ONE    DAMAGE 115%
WeaponBonus = FRENZY_TWO    DAMAGE 130%
WeaponBonus = FRENZY_THREE  DAMAGE 145%
```

**Comment by ZekeDlyoung** [comment]
What if:
```
WeaponBonus = FRENZY_ONE    DAMAGE 130%
;WeaponBonus = FRENZY_TWO    DAMAGE 120%
;WeaponBonus = FRENZY_THREE  DAMAGE 130%

SpecialPower SuperweaponFrenzy
  ReloadTime = 180000

Object Frenzy_InvisibleMarker_Level1
  BonusDuration = 10000
  BonusRange = 200

Object Frenzy_InvisibleMarker_Level2
  BonusDuration = 20000
  BonusRange = 200

Object Frenzy_InvisibleMarker_Level3
  BonusDuration = 30000
  BonusRange = 200
```

Then FRENZY_TWO and FRENZY_THREE can be repurposed for other things 🙃

**Comment by xezon** [comment]
I think it is ok to leave the increments. This is how many Level 3 Powers are setup.

**Comment by ZekeDlyoung** [comment]
I dunno, for me it makes more sense to be the same level of hyped/angry for longer, rather than being angrier/more hyped up for the same amount of time 🤷

**Comment by xezon** [comment]
What would scale is the DAMAGE %, which looks reasonable to me.

Same damage for longer also makes sense.

**Comment by commy2** [comment]
Benefit of a fixed % and expanding duration is that it is visually obvious when and what the buff applies, because there is only one.

**Comment by xezon** [comment]
```
WeaponBonus = FRENZY_ONE    DAMAGE 130%
WeaponBonus = FRENZY_TWO    DAMAGE 130%
WeaponBonus = FRENZY_THREE  DAMAGE 130%

SpecialPower SuperweaponFrenzy
  ReloadTime = 180000

Object Frenzy_InvisibleMarker_Level1
  BonusDuration = 10000
  BonusRange = 200

Object Frenzy_InvisibleMarker_Level2
  BonusDuration = 20000
  BonusRange = 200

Object Frenzy_InvisibleMarker_Level3
  BonusDuration = 30000
  BonusRange = 200
```

**Comment by ExiLe-ZH** [comment]
Level 1 is used a bit competitively, although often for the 'scan ability'. With that being removed the Frenzy might need an upgrade. Cooldown would be my preference.

Also same like the other non-meta abilities, level 2 and 3 cost too many genpoints, regardless of how much you buff it. For all those abilities I would consider removing level 3 and potentially make lvl1=lvl2 & lvl2=lvl3 (or to explain that differently, remove lvl1 so that you start with lvl2 for 1 genpoint). Or custom tweak them.

**Comment by xezon** [comment]
We could also prolong the durations, so that
```
Object Frenzy_InvisibleMarker_Level1
  BonusDuration = 15000

Object Frenzy_InvisibleMarker_Level2
  BonusDuration = 30000

Object Frenzy_InvisibleMarker_Level3
  BonusDuration = 45000
```

or a bit more conservative
```
Object Frenzy_InvisibleMarker_Level1
  BonusDuration = 12000

Object Frenzy_InvisibleMarker_Level2
  BonusDuration = 24000

Object Frenzy_InvisibleMarker_Level3
  BonusDuration = 36000
```

Having level 2 and 3 for so long active could make them really attractive to use.

China units are typically slow, so prolonged bonus times will help.

However, then cooldown timer should not be just 3 minutes I believe.

**Comment by Jundiyy** [comment]
With a 130% bonus, original time might be worth it. 

Then move to 3 minutes if needed.

**Comment by ExiLe-ZH** [comment]
I think increased damage is more useful than the duration, since the ability is activated in big important engagements as they happen, so that the enemy can't retreat anymore. These engagements usually don't last that long but Frenzy can easily be the flipping factor.

Thinking about my earlier idea of only having 2 levels, but maybe instead of removing level 1, remove level 2 so that you can instantly upgrade to level 3 for a grant total of 2 points. If you would start off with level 2, then level 3 might not be attractive enough either.

This way you don't have to tweak values either.

But one thing is for sure, no decent player will spent 3 points. Ever.

**Comment by xezon** [comment]
I think user facing it is best to keep the 3 Frenzy levels. Otherwise it looks like things have been taken away from the General Powers. And players will be sad about it.

Maybe Level 3 would have to be scaled more then to be attractive, such that
Level 1 = 10 seconds
Level 2 = 20 seconds
Level 3 = 40 seconds

This way Level 2 and Level 3 are a 2x upgrade from their predecessor.

Raising Level 1 to 130% DAMAGE has one big consequence: Infantry General has Frenzy 1 available on Rank 1, this means it will greatly benefit from it in early game, which in turn makes it more difficult for other Chinas to compete. Inf Outposts are very strong already even without the Frenzy bonus.

**Comment by ExiLe-ZH** [comment]
> I think user facing it is best to keep the 3 Frenzy levels. Otherwise it looks like things have been taken away from the General Powers. And players will be sad about it.
> 
> 

Yes this makes it hard to make such decision.

> 
> Maybe Level 3 would have to be scaled more then to be attractive, such that
> 
> Level 1 = 10 seconds
> 
> Level 2 = 20 seconds
> 
> Level 3 = 40 seconds
> 
> 
> 
> This way Level 2 and Level 3 are a 2x upgrade from their predecessor.
> 
> 
> 
> Raising Level 1 to 130% DAMAGE has one big consequence: Infantry General has Frenzy 1 available on Rank 1, this means it will greatly benefit from it in early game, which in turn makes it more difficult for other Chinas to compete. Inf Outposts are very strong already even without the Frenzy bonus.

Don't think level 1 needs this big of a buff yea, just level 2 and 3 need improvement.

Maybe go with 110>130>150, think genpoint investment justifies this.

There is one problem with Infantry though, using it on Outposts has zero effect, must evac the infantry. Many people don't know this. But don't think this can be fixed?

**Comment by xezon** [comment]
Hmm that is bad indeed. I assume it simply does not apply to passengers?

**Comment by ExiLe-ZH** [comment]
> Hmm that is bad indeed. I assume it simply does not apply to passengers?

Yes, same as gps scrambler, however here it's logical

**Comment by xezon** [comment]
Ok, with that problem in mind, the Level 1 Frenzy for Infantry General is less powerful.

**Comment by MTKing4** [comment]
3 levels of (frenzy/Emergency Repair) is really pain in the ass to fix, I can't imagine how anyone would spend more than 1 point on this over any other gen point

In fact the only time when you spend 1 on it is when you have an extra point and don't know what to do with it.

The only way this changes is by making it so good that you take it instead of artillery, keep in mind, this WILL change the meta by a lot if achieved.

imagine having a build just to get (frenzy/emergency Repair) and leveled up, it has to be game changing to be attractive for players

**Comment by MTKing4** [comment]
Possible useful Buffs for Frenzy, 

Make cooldown 1 minute
Maybe even Less

Keep same damage and range

I think it needs to be affecting engagements a lot to be worth it to get higher levels

**Comment by Jundiyy** [comment]
I think 130% with 40s of bonus is quite good.

**Comment by FDPUser443** [comment]
Proposal 5 ?
Add attack speed bonus 
Example:
L1: +10%
L2: +20%
L3  +35%

**Comment by fastAiRR** [comment]
I used to pick frenzy alot, and use it on my migs as a kind of signature lol. Anyways, its cool it makes them red, but it doesnt materially change anything, still need 4 to make a firestorm, 3 would be better with frenzy. 
Its a small almost irrelevant thing, but its not right at the moment. 

Muff gets frenzy also, and others use it for scan. I wouldn't remove the scan ability, that's kind of cool and seems intentional to  me. Kind of like an easter egg

**Comment by Stubbjax** [comment]
> I wouldn't remove the scan ability, that's kind of cool and seems intentional to me. Kind of like an easter egg

Why would it be intentional? It is counterintuitive and detracts from USA's Satellite Scan ability. Should Emergency Repair reveal the area too?

Not only that, but it creates a balance issue where Infantry General can determine the opponent's faction while other Chinas cannot.

**Comment by xezon** [comment]
We already discussed Frenzy Scan and we agreed to remove it.
* #591

**Comment by fastAiRR** [comment]
If you have decided already then ok. But IMO its an easter egg left by the game developers. You guys are trying to find ways of making people use frenzy more, and your main plan is to remove its scan ability? That's one of the main reasons people pick it. 
I'm not trying to criticise, but its probably changes like this people will not like, particularly as only high level players use this ability, and those are surely the ones you want to champion this patch.

Maybe there was along discussion I missed, but removing its scan ability is not going to increase its appeal but diminish it.

**Comment by xezon** [comment]
There is no evidence that this is intentional or an Easter egg. Game definitions are copied from one place to another which means misplaced logic remnants are no rare occurrence in INI files.

As for this Superweapon Ability, I think it is the wrong focus to make an ability attractive by something it is not advertised for. Players should pick Frenzy for the limited power increase of select forces, not the ability to reveal the shroud for a little while.

Despite, the Level 1 Frenzy Scan is mainly useful for Infantry General. Other China's do not bank as much on Level 3 Frenzy Scan. As it stands, Infantry General is one of the best factions already. It needs no such tricks.

**Comment by commy2** [comment]
War Chant is a comparable "point and buff units in circle temporarily" ability in BFME that, in contrast to Frenzy, is considered very good.

- It has a smaller radius, 100 instead of 200, but BFME hordes are large, so you can roughly fit your entire army inside either way (it is enough to touch 1 unit to buff the entire horde of 10)
- It is available as first ability for 1 point (you start with 1 point), so it is essenitally a rank 1 ability (like it already is for Infantry General).
- It gives + 50% damage AND +50% armor, compared to just 10,20 or 30% damage
- It lasts for 1 minute instead of just 10, 20 or 30 seconds
- It reacharges in 2 minutes and 15 seconds = 135 seconds, instead of in 3 minutes = 180 seconds

So in summary, War Chant is a bigger buff that is available earlier (aside for Infgen), cheaper or equally as cheap (1 point vs 1 to 3 points), lasts longer and recharges faster. It is better in every way. 

Additionally, Frenzy competes with abilities that can just wipe entire buildings like Carpet Bombing or Artillery Barrage, stuff that BFME doesn't have aside from Balrog or Army of the Dead perhaps.

In conclusion, Frenzy would have to be drastically buffed in multiple ways to be considered competetive.

**Comment by xezon** [comment]
Interesting that War Chant is that much better, but then of course it is a different game.

**Comment by MTKing4** [comment]
I like that, make frenzy equal to war chant :D

**Comment by commy2** [comment]
War Chant effective total boost is +150% (+ 50% damage, +100% health equivalent), so it would be slightly better upgrade than the +140% (+120% damage and +20% health) of a 1.04 vet 1 Quad, except temporary.

**Comment by kABUSE1** [comment]
Bonus range would make this very powerful against humvees, as your gats suddenly become an actual threat for them. I am not 100% sure if that is good. While I hate humvees, this would also feel counter-intuitive to fight against and would introduce another impactful gameplay-change.

I like commy's suggestion the most. Make it 30sec regardless of level and reduce cooldown.
